src/_nebari/upgrade.py

```diff
@@ -705,6 +705,20 @@ def _version_specific_upgrade(
         return config
 
 
+class Upgrade_2024_2_1(UpgradeStep):
+    version = "2024.2.1"
+
+    def _version_specific_upgrade(
+        self, config, start_version, config_filename: Path, *args, **kwargs
+    ):
+        rich.print("\n ⚠️  Warning ⚠️")
+        rich.print(
+            "-> Please run the [green]rm -rf stages[/green] so that we can regenerate an updated set of Terraform scripts for your deployment."
```

Comment:
This can cause trouble for people using local tf state. See https://github.com/nebari-dev/nebari/issues/2271. I think just runnning `nebari render` should do the same thing here. Can we verify this is actually needed?

Reply:
At a minimum I think we should change this to `mv stages stages.old` or something similar.  This messed me up recently.  Luckily I was able to get stages back by trash-cli